homeassistant/components/bzutech/sensor.py

```diff
@@ -0,0 +1,331 @@
+"""Sensor for BZUTech integration."""
+from dataclasses import dataclass
+from datetime import date, datetime, timedelta  # noqa: D100
+from decimal import Decimal
+import logging
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+    CONCENTRATION_PARTS_PER_MILLION,
+    LIGHT_LUX,
+    PERCENTAGE,
+    SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+    UnitOfElectricCurrent,
+    UnitOfElectricPotential,
+    UnitOfInformation,
+    UnitOfSoundPressure,
+    UnitOfTemperature,
+    UnitOfTime,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from . import BzuCloudCoordinator
+from .const import CONF_CHIPID, DOMAIN
+
+
+@dataclass(frozen=True)
+class BzuSensorEntityDescription(SensorEntityDescription):
+    """Describe bzu sensor entity."""
```

Comment:
if you're adding nothing to this, you can just use the normal `SensorEntityDescription` instead